warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -489,6 +498,40 @@ public void run() {
         log.info("Bulk map file loader shutting down.");
     }
     
+    // take ownership by renaming files and then running cleanup script
+    private void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
+        for (int i = 0; i < cleanupDirectories.length; i++) {
+            boolean success = false;
+            
+            try {
+                success = getFileSystem(srcHdfs).rename(new Path(cleanupDirectories[i], CLEANUP_FILE_MARKER),
+                                new Path(cleanupDirectories[i], REMOVE_FILE_MARKER));
+                log.info("Renamed " + cleanupDirectories[i] + '/' + CLEANUP_FILE_MARKER + " to " + REMOVE_FILE_MARKER);
+            } catch (IOException e2) {
+                log.error("Exception while marking " + cleanupDirectories[i] + " for removal: " + e2.getMessage(), e2);
+            }
+            
+            if (success) {
+                markSourceFilesLoaded(cleanupDirectories[i]);
+                getFileSystem(srcHdfs).delete(cleanupDirectories[i], true);
+                runCleanUpScript(cleanupDirectories[i], true);
+            }
+        }
+        
+        Path[] failedDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + LOADING_FILE_MARKER));
```

Comment:
If we start an additional loader later on, there will be jobs already in loading.  Best to handle this case from the start script after we verify there are no loaders currently running.